packages/terraform-modules/src/base/ApplicationLambdaCodeDeploy.ts

```diff
@@ -43,7 +44,19 @@ export class ApplicationLambdaCodeDeploy extends Construct {
   ) {
     super(scope, name);
 
-    this.codeDeployApp = this.setupCodeDeploy();
+    const { codeDeployApp, codeDeployGroup } = this.setupCodeDeploy();
+    this.codeDeployApp = codeDeployApp;
+    this.codeDeployGroup = codeDeployGroup;
+
+    new TerraformOutput(this, 'lambda_codedeploy_app', {
```

Comment:
Start outputing the variables for us to use if we wanted.

However, using them will be a seperate future followup. Because we use multiple lambdas in each CI we can't use a "static" id like we do in ECS. So need to think through the best way to dynamically pull the right ones.

Reply:
can you expand on what you mean by "multiple lambdas in each CI"?